examples/car_control/run.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+
+import numpy as np
+import os
+import spear
+import time
+
+common_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "common"))
+import sys
+
+sys.path.append(common_dir)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--actions_file", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "actions.csv")))
+    parser.add_argument("--bp_class_reference", default="/Game/VehicleTemplate/Blueprints/OffroadCar/OffroadCar_Pawn.OffroadCar_Pawn_C")
+    args = parser.parse_args()
+
+    np.set_printoptions(linewidth=200)
+
+    # load config
+    config = spear.get_config(
+        user_config_files=[
+            os.path.realpath(os.path.join(common_dir, "default_config.common.yaml")),
+            os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
+
+    spear.configure_system(config)
+    instance = spear.Instance(config)
+
+    instance.engine_service.begin_tick()
+
+    gameplay_statics_class = instance.unreal_service.get_static_class(class_name="UGameplayStatics")
+    gameplay_statics_default_object = instance.unreal_service.get_default_object(uclass=gameplay_statics_class, create_if_needed=False)
+
+    set_game_paused_func = instance.unreal_service.find_function_by_name(uclass=gameplay_statics_class, name="SetGamePaused")
+
+    get_player_controller_func = instance.unreal_service.find_function_by_name(uclass=gameplay_statics_class, name="GetPlayerController")
+    get_player_controller_return_values = instance.unreal_service.call_function(uobject=gameplay_statics_default_object, ufunction=get_player_controller_func,
+                                                                                args={"PlayerIndex": 0})
+    player_controller = instance.unreal_service.to_handle(get_player_controller_return_values["ReturnValue"])
+    controller_class = instance.unreal_service.load_class(class_name="UObject", outer=0, name="/Script/Engine.Controller", filename="")
+
+    enhanced_input_component_class = instance.unreal_service.load_class(class_name="UObject", outer=0, name="/Script/EnhancedInput.EnhancedInputComponent", filename="")
+    enhanced_input_component = instance.unreal_service.get_component_by_class(player_controller, enhanced_input_component_class)
```

Comment:
I think it's cleaner to `get_component_by_name` or `get_component_by_type` rather than doing all of these `load_class` ops. We can register additional types if necessary.

Reply:
Temparary experimental code, removed.